webapp/src/toolboxeditor.tsx

```diff
@@ -118,7 +119,7 @@ export abstract class ToolboxEditor extends srceditor.Editor {
         let that = this;
 
         function filterNamespaces(namespaces: [string, pxtc.CommentAttrs][]) {
-            return namespaces.filter(([, md]) => !md.deprecated && (isAdvanced ? md.advanced : !md.advanced));
+            return namespaces.filter(([, md]) => !(md.deprecated && that.parent.state.tutorialOptions == undefined)&& (isAdvanced ? md.advanced : !md.advanced));
```

Comment:
I think this should maybe just get pulled into a method, i.e. `this.parent.inTutorial()`

Reply:
totally, I'll do that!